src/looplets/lookups.jl

```diff
@@ -37,7 +37,9 @@ function lower(root::FinchNode, ctx::AbstractCompiler,  ::LookupStyle)
             open_scope(body_3, ctx_2)
         end
         @assert isvirtual(root.ext)
-        if query(call(==, measure(root.ext.val), 1), ctx)
+
+        target = is_continuous_extent(root.ext) ? 0 : 1
+        if query(call(==, measure(root.ext.val), target), ctx)
```

Comment:
`measure(ext)` is defined differently depending on the type of `ext`. 

```
measure(ext::Extent) = call(+, call(-, ext.stop, ext.start), 1)
measure(ext::ContinuousExtent) = call(-, ext.stop, ext.start) 
```

Reply:
does this make sense? I sortof think it does, but it's different than what we thought.